index.html

```diff
@@ -77,6 +77,39 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- make sure there is one line of space above and below your card -->
 
+
+<!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Yurii Zbitniev</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="www.twitter.com" target="_blank">Your handle</a>
+          </p>
+          <p class="about">about you</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="#" target="_blank" title="First Resource">
+                  Resource 1
+                </a>
+              </li>
+              <li>
+                <a href="#" target="_blank" title="Second Resource">
+                  Resource 2
+                </a>
+              </li>
+              <li>
+                <a href="#" target="_blank" title="Third resource">
+                  Resource 3
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- ________*END TEMPLATE* Contributor card END ________  -->
+
```

Comment:
You only need to leave a single blank line in between the template and your (others') code.